services/ui-src/src/utils/validation/schemas.ts

```diff
@@ -10,24 +10,16 @@ import { validationErrors as error } from "verbiage/errors";
 import { Choice } from "types";
 
 // TEXT - Helpers
-const testForEmptyValue = (value: any) => {
```

Comment:
Note: This method has a bug in it where when you have a value that is equal to `""`, it gets caught by line 14 because `""` is falsy. This should work better now.